test/fuzzing/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
I have noticed that every cmake file of yours has a leading empty line. Is there a reason for this? 😉

Reply:
Not really. I sometimes have license notice here...